index.html

```diff
@@ -17,36 +17,42 @@
             <li class="navbar-toggle">
               <img src="img/Union.png" alt="menubar" />
             </li>
+           <nav class="desktopnav">
+            <li class="portfolio" id="portfoliotxt">Portoflio</li>
+            <li class="about" id="abouttxt">About</li>
+            <li class="contactmenu" id="contactmenutxt">Contact</li>
+          </nav>
           </nav>
         </ul>
       </header>
       <section class="container">
-        <main>
-          <h1 class="headerone font">I’m Emmanuel Glad to see you!</h1>
+        <section class="maincontainer">
+        <main class="maintxt">
+          <h1 class="headerone font">I’m Emmanuel<br> Glad to see you!</h1>
           <p class="text font">
             I’m a software developer! I can help you build a product , feature
             or website Look through some of my work and experience! If you like
             what you see and have a project you need coded, don’t hestiate to
             contact me.
           </p>
-        </main>
-        <footer>
           <p class="connect font">LET’S CONNECT</p>
+        </main>
           <ul class="social">
-            <li><img src="img/Twitter.png" alt="" /></li>
-            <li><img src="img/Linkedin icon.png" alt="" /></li>
-            <li><img src="img/MVector.png" alt="" /></li>
-            <li><img src="img/Github.png" alt="" /></li>
-            <li><img src="img/PeaceVector.png" alt="" /></li>
+            <li><a href="#"><img src="img/Twitter.png" alt="" /></a></li>
+            <li><a href="#"><img src="img/Linkedin icon.png" alt="" /></a></li>
+            <li><a href="#"><img src="img/MVector.png" alt="" /></a></li>
+            <li><a href="#"><img src="img/Github.png" alt="" /></a></li>
+            <li><a href=""><img src="img/PeaceVector.png" alt="" /></a></li>
           </ul>
-        </footer>
       </section>
     </section>
```

Comment:
- [ ] same as on the mobile version kindly consider adding that border-left radius and also  fix the padding on the H1 to implement well the Figma design 
- your design  👇 
![image](https://user-images.githubusercontent.com/60146030/221366248-f672d4be-38e2-473e-899d-bff86b02b20c.png)
- Figma design 👇 
![was](https://user-images.githubusercontent.com/60146030/221366360-99747fce-9b94-438a-a8e8-8ead0b96eab3.PNG)

Reply:
Done.

Reply:
done